code/modules/jobs/job_types/paramedic.dm

```diff
@@ -21,6 +21,15 @@
 
 	family_heirlooms = list(/obj/item/storage/firstaid/ancient/heirloom)
 
+	mail_goodies = list(
+		/obj/item/reagent_containers/hypospray/medipen = 20,
+		/obj/item/reagent_containers/hypospray/medipen/oxandrolone = 10,
+		/obj/item/reagent_containers/hypospray/medipen/salacid = 10,
+		/obj/item/reagent_containers/hypospray/medipen/salbutamol = 10,
+		/obj/item/reagent_containers/hypospray/medipen/penacid = 10,
+		/obj/item/reagent_containers/hypospray/medipen/survival/luxury = 5
```

Comment:
a proper pinpointer, an advanced medkit, and a bluespace body bag might also be nice additions to the pool